lib/src/model/scan_mode.dart

```diff
@@ -14,6 +14,7 @@ enum ScanMode {
   lowLatency,
 }
 
+/// Converts scanmode to integer representation.
```

Comment:
What is "scanmode"? Either "[scanMode]" (ID reference or at lease `code` snippet), or "scan mode" (natural reference), please.